accounts/accounts.go

```diff
@@ -30,6 +30,8 @@ import (
 	"golang.org/x/crypto/sha3"
 )
 
+var BLSHardwareWalletAddress = common.HexToAddress("0x0000000000000000000000000000000000000001")
```

Comment:
Any way to use `cfg.BLSbase` so this only has to be hardcoded once?

Reply:
There are places where `BLSHardwareWalletAddress` is used by an arbitrary `BLSbase` wouldn't make sense. I changed the line in `cmd/utils/flags.go` to use this constant instead so it's only hardcoded once